src/cloud-api-adaptor/cmd/agent-protocol-forwarder/main.go

```diff
@@ -34,7 +34,9 @@ var logger = log.New(log.Writer(), "[forwarder] ", log.LstdFlags|log.Lmsgprefix)
 type Config struct {
 	tlsConfig           *tlsutil.TLSConfig
 	daemonConfig        daemon.Config
```

Comment:
@davidhadas can you please split the apf code changes introducing APF config file and the test related code changes into separate PRs.  Also wondering why APF config is required separate and the daemonConfig cannot be used ?

Reply:
Please elaborate. 
We already have an e2e test merged for daemonConfig in which the keys are delivered using the daemonConfig. 
This test suppose to cover a case where the daemonConfig is untrusted and we use Trustee to deliver the keys. 
 
 This work is on hold until there is a decision to go ahead with this additional e2e test. 
 I have rebased it, but it is currently untested and should not be merged.